doc/deployment_guide.md

```diff
@@ -49,5422 +49,6052 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
-> * Logging
-> * Example of sample_csp_profile.ini
-> 
-
-
-#### Hugepages overview
-
-The Dell EMC Ready Architecture for Red Hat OpenStack Platform v13.0 provides the ability to enable hugepage support on all Nova compute nodes in the solution.
-
-The Linux kernel partitions memory into basic units called pages. The default number of pages is 49152 in the x86 architecture. Hugepages allow the Linux kernel to utilize the multiple page size capabilities of modern hardware architecture.
-
-Earlier, in previous release, fixed values were used for the number of hugepages. The number of hugepages was set to 49152 and 96 when the hugepage size was set to 2MB and 1GB respectively.
-
-In Dell EMC Ready Architecture for Red Hat OpenStack Platform Software Deployment Guide - Version 13, the number of hugepages is computed during runtime. The OpenStack Ironic API is used to fetch the **total memory size** of a compute node, and the following formula is applied to calculate the number of hugepages:
-
-
-> Hugepage number = ( Total Memory Size - ( memory reserved for host OS + memory reserved for kernel)) / size of a hugepage
-
-The number of hugepages is calculated for each compute node based on that nodes memory size. The smallest number of hugepages supported by all compute nodes is then used to configure all the compute nodes with this number of hugepages. This means that all compute nodes will be configured with the number of hugepages that are supported by the compute node with the least amount of memory.
-
-#### Prerequisites
-
-The following requirements must be met before enabling hugepages:
-
-1. Memory size must be greater than or equal to 128GB on every compute node.
-
-   It is assumed that:
-   
-   - The memory reserved for the host OS = 12GB
-   - The memory reserved for the kernel = 4GB
-
-2. Refer to the automated or manual deployment sections before setting these values.
-
-
-#### Enabling and deploying hugepages
-This section provides the instructions to configure and deploy hugepage support.
-
-
-#### Enabling hugepages
-
-HugePages configuration parameters
-
-```
-hpg_enable: (true/false) (Set "true" for enabling hugepages and "false" for disabling it.)
-hpg_size: 1GB/2MB (The hugepage size to be configured on every compute node. The default is: 1GB)
-```
 
-**Follow the procedure below to enable Hugepages on Dell EMC Ready Architecture for Red Hat OpenStack Platform version 13.0:**
-
-1.	Open an SSH session to the SAH node.
-2.	Log in as the root user.
-3.	Refer to section Automation configuration files in this document.
-4.	Set hpg_enable=true
-5.	Set hpg_size to the size of the hugepages to use
-	
-	> Note: Valid values are 1GB and 2MB
-
-
-#### Deploying with hugepages
-
-After enabling hugepages in the hardware stamp's .ini file, perform the following steps to deploy hugepages in Dell EMC Ready Architecture for Red Hat OpenStack Platform.
-
-
-1.	Open an SSH session to the SAH node.
-2.	Run the following commands to deplo
```

Comment:
Change the diagram to represent OS10 interfaces, not cumulus interfaces. i.e swp should be changed to ethernet